CONTRIBUTING.md

```diff
@@ -13,6 +13,13 @@ are always appreciated!
 You can use Docker and docker-compose to test shuttle locally during development. See the [Docker install](https://docs.docker.com/get-docker/)
 and [docker-compose install](https://docs.docker.com/compose/install/) instructions if you do not have them installed already.
 
+```bash
+git clone git@github.com:shuttle-hq/shuttle.git
+cd shuttle
+git submodule init
+git submodule update
```

Comment:
Nice! Maybe only these two lines should be in a section/heading about checking out the examples submodule?

Reply:
I move this close to `examples` section :)